changelog/fragments/added-suppress-override-values-flag-to-helm-operator.yaml

```diff
@@ -0,0 +1,54 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Currently, when setting `overrideValues` in your `watches.yaml`, every entry is logged with its key and its value.
+      When working with confidential information such as credentials, logging those information would not be optimal nor
+      desirable. Therefore, there should be a way to silence/suppress these values.
+      
+      This feature introduces the `--suppress-override-values` boolean-flag for the `helm-operator`. When set to true,
+      the value of the overrideValues entry is sanitized with this value "****".
+      
+      This is a simple example of `overrideValues` of the `watches.yaml`
+      ```yaml
+      overrideValues:
+        x: y
+      ```
+      
+      The default-value of `--suppress-override-values` is `false` so therefore we would get this warning logged:
+      `Chart value x overridden to y by operator's watches.yaml`
+
+      When setting `--suppress-override-values` `true` the warning will be logged in the following manner:
+      `Chart value x overridden to **** by operator's watches.yaml`
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: Header text for the migration section
+      body: |
+        Body of the migration section. This should be formatted as markdown and can
+        span multiple lines.
+
+        Using the YAML string '|' operator means that newlines in this string will
+        be honored and interpretted as newlines in the rendered markdown.
```

Comment:
Can probably remove this section. I don't think any migration guide is needed for this change